lib/chef/user.rb

```diff
@@ -144,12 +287,17 @@ class << self
     end
 
     def self.list(inflate=false)
-      response = Chef::REST.new(Chef::Config[:chef_server_url]).get_rest('users')
-      users = if response.is_a?(Array)
```

Comment:
We should at least figure out what happens if you run the new `knife user create` against CS 11. There's a chance the HTTP request will just work, in which case a little bit more effort should get us to a decent place compatibility-wise. If it doesn't work, then we should handle that error and be very clear about what the user needs to do instead.